jquery.kinetic.js

```diff
@@ -226,6 +224,12 @@
         if (self._useTarget(e.target, e) && self.elementFocused){
           return false;
         }
+      },
+      // prevent selection when dragging
+      selectStart: function (e){
+        if (self._useTarget(e.target, e)){
```

Comment:
what I want to do is like this:
http://jsfiddle.net/tsaikd/4kL3tc5o/
some text are selectable inside

if the selectStart behavior doesn't make sense, how about add a new flag to control?